<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassGen</string>
  </property>
  <property name="windowOpacity">
   <double>-1.000000000000000</double>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_gen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QListView" name="lst_pswd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>361</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckbx_abc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckbx_ABC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ABC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckbx_spec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/*-+</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckbx_123">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QSlider" name="sl_qnt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_qnt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
